cmd/gctcli/helpers.go

```diff
@@ -45,9 +45,7 @@ func closeConn(conn *grpc.ClientConn, cancel context.CancelFunc) {
 // to allow for proper use at RPCServer
 func negateLocalOffset(t time.Time) string {
 	_, offset := time.Now().Zone()
-	loc := time.FixedZone("", -offset)
-
-	return t.In(loc).Format(common.SimpleTimeFormat)
+	return t.Add(-time.Duration(offset) * time.Second).Format(common.SimpleTimeFormat)
```

Comment:
How about this?

```suggestion
	return t.In(time.UTC).Format(common.SimpleTimeFormat)
}

```

Reply:
I could have sworn that there was a reason why we didn't do this........ But its matching
cmd:
```
2022-12-23 15:59:00 +1100 AEDT
```
rpcserver
```
 2022-12-23 15:59:00 +1100 AEDT
```

# 🌞